static/app/views/settings/projectSourceMaps/projectSourceMaps.tsx

```diff
@@ -38,58 +38,85 @@ enum SORT_BY {
   DESC = '-date_added',
 }
 
+enum SourceMapsBundleType {
+  Release,
+  DebugId,
+}
+
 function SourceMapsTableRow({
+  bundleType,
   onDelete,
   name,
   fileCount,
   link,
   date,
   idColumnDetails,
 }: {
+  bundleType: SourceMapsBundleType;
   date: string;
   fileCount: number;
   link: string;
   name: string;
   onDelete: (name: string) => void;
   idColumnDetails?: React.ReactNode;
 }) {
+  const isEmptyReleaseBundle =
+    bundleType === SourceMapsBundleType.Release && fileCount === 0;
```

Comment:
```suggestion
    bundleType === SourceMapsBundleType.Release && fileCount === -1;
```

We use the sentinel value `-1` for signaling absence of bundle. `0` means bundle present but empty.